crates/noirc_evaluator/src/ssa_refactor/acir_gen/mod.rs

```diff
@@ -115,6 +115,12 @@ impl Context {
                 assert_eq!(result_ids.len(), 1, "Cast ops have a single result");
                 self.ssa_value_to_acir_var.insert(result_ids[0], result_acir_var);
             }
+            Instruction::Not(value_id) => {
+                let result_acir_var = self.convert_ssa_not(*value_id, dfg);
+                let result_ids = dfg.instruction_results(instruction_id);
+                assert_eq!(result_ids.len(), 1, "Not ops have a single result");
+                self.ssa_value_to_acir_var.insert(result_ids[0], result_acir_var);
```

Comment:
when #1436 merges, this will need to change

Reply:
pulled and resolved